tests/postgres/derives.rs

```diff
@@ -58,6 +58,14 @@ enum ColorUpper {
     Blue,
 }
 
+#[derive(PartialEq, Debug, sqlx::Type)]
+#[sqlx(rename = "color_screaming_snake")]
+#[sqlx(rename_all = "screaming_snake_case")]
```

Comment:
Looks like the case for this and line 54 need to be updated.

Reply:
Ah yup, my bad for missing this!